tasks/buildsources.js

```diff
@@ -40,50 +53,66 @@ module.exports = function(grunt) {
 		}
 		scanDir(polyfillSourceFolder);
 
-		dirs.forEach(function(polyfillPath) {
+		var promises = dirs.map(function(polyfillPath) {
 			var config;
 			var configPath = path.join(polyfillPath, 'config.json');
 			var detectPath = path.join(polyfillPath, 'detect.js');
 			var polyfillSourcePath = path.join(polyfillPath, 'polyfill.js');
 			var featureName = polyfillPath.substr(polyfillSourceFolder.length+1).replace(/\//g, '.');
 
+			// Load the polyfill's configuration
 			try {
+				if (!fs.existsSync(configPath)) return;
+				config = JSON.parse(fs.readFileSync(configPath));
+				config.baseDir = path.relative(path.join(__dirname,'../polyfills'), polyfillPath);
+			} catch (e) {
+				throw {name:"Invalid config", message:"Unable to read config from "+configPath};
+			}
 
-				// Load the polyfill's configuration
-				try {
-					if (!fs.existsSync(configPath)) return;
-					config = JSON.parse(fs.readFileSync(configPath));
-					config.baseDir = path.relative(path.join(__dirname,'../polyfills'), polyfillPath);
-				} catch (e) {
-					throw {name:"Invalid config", message:"Unable to read config from "+configPath};
-				}
+			config.hasTests = fs.existsSync(path.join(polyfillPath, 'tests.js'));
 
-				config.hasTests = fs.existsSync(path.join(polyfillPath, 'tests.js'));
+			if (config.licence) {
+				throw 'Incorrect spelling of license property in '+featureName;
+			}
 
-				if (config.licence) {
-					throw 'Incorrect spelling of license property in '+featureName;
-				}
+			if (config.build && config.build.commonjs === true) {
+				config.rawSource = streamToString(browserify().add(polyfillSourcePath).bundle());
```

Comment:
I mean `browserify` should be called on the next step, for a bundle for a browser, not for files from this directory. Your approach will work for 1 polyfill from `core-js`, but if you wanna use no only one - you will have a bundle of size in megabytes because of duplication internal modules.